js/listdates.js

```diff
@@ -1,87 +1,58 @@
 /* global moment, humanizeDuration */
 var content = "";
```

Comment:
If you don't have domready, then any scripts that act on html MUST be loaded at the bottom of the body. Not sure if async=true would help or not. There may be other reasons, but I'm not sure.